packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -21,12 +21,13 @@ import {
   useValidatorState,
 } from '../../lib/validator';
 import { formatCurrencyInCents } from '../../lib/formats';
+import { AppContext } from '../../lib/AppContext';
 
 import './index.scss';
 import { Plan } from '../../store/types';
 
 // ref: https://stripe.com/docs/stripe-js/reference#the-elements-object
-const STRIPE_ELEMENT_STYLES = {
+let stripeElementStyles = {
```

Comment:
I don't think this needs to be switched from const to let - the variable itself isn't reassigned, just properties of the object.